admin/src/components/Button/Index.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx";
+import { ComponentPropsWithoutRef } from "react";
+import Spinner from "../Spinner";
+
+interface ButtonProps extends ComponentPropsWithoutRef<"button"> {
+  loading?: boolean;
+  styledClassName?: string;
```

Comment:
i reeeeally like to use default prop name like extending the component instead of creating a new api for it, you can add changes regarding my comments in a new PR if you agree with them